buildpipeline/Core-Setup-Windows-Arm-BT.json

```diff
@@ -62,6 +62,7 @@
       "alwaysRun": false,
       "displayName": "Install Signing Plugin",
       "timeoutInMinutes": 0,
+      "condition": "and(succeeded(), in(variables.PB_SignType, 'real', 'test'))",
```

Comment:
Later in this file the type is hard-coded to `real`, is this ok or should it use `$(PB_SignType)`?

https://github.com/weshaggard/core-setup/blob/75bed989ccb49da15048108cf1fe8ac379c3e461/buildpipeline/Core-Setup-Windows-Arm-BT.json#L65-L72

Reply:
That is OK and is coming from the microbuild task. We ultimately override it by passing in SignType to our build command so it is never really used.